src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -64,8 +64,16 @@
     // 'SQL' represents SQL Server, while 'SQLAzure' represents SQL Azure.
     public static final String SERVER_TYPE_SQL_SERVER = "SQL";
     public static final String SERVER_TYPE_SQL_AZURE = "SQLAzure";
+    
     // private static SqlType types = null;
     private static ArrayList<SqlType> types = null;
+    private final static int ENGINE_EDITION_FOR_SQL_AZURE = 5;
+    private final static int ENGINE_EDITION_FOR_SQL_AZURE_DW = 6;
+
+    // whether we determined if the target server is SQL Azure or DW
+    private static boolean _determinedSqlAzureOrSqlServer = false;
+    private static boolean _isSqlAzure = false;
+    private static boolean _isSqlAzureDW = false;
```

Comment:
Let's use the same methods to check for Azure DB/DW as in PR #903, implemented in AbstractTest class.

Reply:
manually merged

Reply:
manually merged for now